hasura/metadata/databases/default/tables/public_profiles.yaml

```diff
@@ -79,23 +79,23 @@ update_permissions:
         _eq: X-Hasura-User-Id
     set:
       id: X-Hasura-User-Id
-  role: user
+  role: superadmin
```

Comment:
I don't really understand why this changed, especially the indent?

Reply:
might be a previous merge that caused the indenting to go off